src/Text/Pandoc/Readers/LaTeX.hs

```diff
@@ -414,6 +416,12 @@ blockCommands = M.fromList $
   , "pagebreak"
   ]
 
+blockTextcolor :: PandocMonad m => LP m Blocks
+blockTextcolor = do
+    skipopts 
+    color <- braced
+    divWith ("",[],[("style","color: " ++ color)]) <$> grouped block <* notFollowedBy inline
+
```

Comment:
PS. I used {} in inlineContents, because I can't indent in this text box, but you should use indentation probably.